src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/RequireThisCheck.java

```diff
@@ -1303,29 +1346,29 @@ public DetailAST getFrameNameIdent() {
         /**
          * Check whether the frame contains a field or a variable with the given name.
          *
-         * @param nameToFind the IDENT ast of the name we're looking for.
+         * @param identToFind the IDENT ast of the name we're looking for.
          * @return whether it was found.
          */
-        protected boolean containsFieldOrVariable(DetailAST nameToFind) {
-            return containsFieldOrVariableDef(varIdents, nameToFind);
+        protected boolean containsFieldOrVariable(DetailAST identToFind) {
+            return containsFieldOrVariableDef(varIdents, identToFind);
         }
 
         /**
          * Check whether the frame contains a given name.
          *
-         * @param nameToFind IDENT ast of the name we're looking for.
+         * @param identToFind IDENT ast of the name we're looking for.
          * @param lookForMethod whether we are looking for a method name.
          * @return whether it was found.
          */
-        protected AbstractFrame getIfContains(DetailAST nameToFind, boolean lookForMethod) {
+        protected AbstractFrame getIfContains(DetailAST identToFind, boolean lookForMethod) {
```

Comment:
Please move all such renaming in separate commit of this PR, message "minor: renaming to make it clear"
We try our best to not mix functional changes and refactoring.

Please move change for method creation to avoid statement count violation, in separate commit also - "minor: move code to method to avoid violation from zxxx in next commit"

Reply:
Done